ivy/functional/backends/paddle/layers.py

```diff
@@ -118,8 +118,22 @@ def conv2d(
     data_format: str = "NHWC",
     dilations: Union[int, Tuple[int, int]] = 1,
     out: Optional[paddle.Tensor] = None,
-) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+):
+    df = "NHWC"
+    if data_format == "NCHW":
+        df = "NCHW"
```

Comment:
I still don't see a need for this, what does this snipped add?

Reply:
yeah, its nit needed